### PR TITLE
Fix for the doubly-periodic supercell thunderstorm case 17

### PR DESCRIPTION
**Pull request by JosephMouallem**
**Description**

Fix for the idealized test case 17 to simulate a supercell thunderstorm in a doubly periodic domain

**How Has This Been Tested?**

Tested on Orion under SHiELD with the latest FMS and physics

**Checklist:**

Please check all whether they apply or not
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by lharris4** [comment]
Hi, Joseph. This looks good. Thanks for taking care of this.

Lucas

**Review by lharris4 — APPROVED** [review]
Looks good to me.

File: tools/test_cases.F90 [hunk 1]
```diff
@@ -5566,7 +5570,8 @@ end subroutine superK_u
 
 
  subroutine SuperCell_Sounding(km, ps, pk1, tp, qp)
- use gfdl_cloud_microphys_mod, only: wqsat_moist, qsmith_init, qs_blend
+ use gfdl_cloud_microphys_mod, only: qsmith_init, qs_blend
+ use gfdl_mp_mod, only: wqsat_moist
```
Comment (bensonr) [inline tools/test_cases.F90:5574]: I think we need to be careful about which one we use.  As I understand it, one is to be compiled with -r8 (physics) and the other with -r4 (inline dycore physics) - but maybe I've got it all wrong.

**Comment by lharris4** [comment]
I believe that this code compiles fine, although perhaps qsmith_init and
qs_blend from gfdl_mp_mod should be used instead.

**Review by lharris4 — APPROVED** [review]
This is an appropriate update for this test. I approve this MR. Thanks for taking care of this.

**Review by bensonr — APPROVED** [review]

**Comment by JosephMouallem** [comment]
Will be part of the upcoming public release, so I will close it.

**Pull request closed without merging**